my-app/src/app/core/core.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeaderComponent } from './header/header.component';
+import { FooterComponent } from './footer/footer.component';
+import { BreadcrumbsComponent } from './breadcrumbs/breadcrumbs.component';
+import { FakeLogoComponent } from './header/fake-logo/fake-logo.component';
+
+
+
+@NgModule({
+  declarations: [HeaderComponent, FooterComponent, BreadcrumbsComponent, FakeLogoComponent],
```

Comment:
please format, you can automate that task using husky and prettier (there are a few `ng add` schematics that could do the setup for you)